coffee/base.py

```diff
@@ -663,6 +731,24 @@ def gencode(self, not_scope=False):
                                                         self.children[0].gencode())
 
 
+class Ifthen(Statement):
+    """ If statement without else branch.
+
+    :param cond: the condition of the statement
+    :param then_body: then branch code
+    """
+
+    def __init__(self, cond, then_body):
+        # If the then branch is a plain list, cast it to a Block.
+        if not isinstance(then_body, Node):
+            then_body = Block(then_body, open_scope=True)
+        super(Ifthen, self).__init__([then_body])
+        self.cond = cond
+
+    def gencode(self, scope=True):
+        return "if (" + str(self.cond) + ")\n" + indent(self.children[0].gencode())
+
+
```

Comment:
Initially there was no IF and I added an If with only the then branch present. In the meantime another If was added with both then and else branches. Do you want me to code this IfThen into the If node?